includes/RemovePIIJob.php

```diff
@@ -72,20 +72,20 @@ public function run() {
 			'cu_changes' => [
 				[
 					'where' => [
-						'cuc_user' => $userId
+						'cuc_actor' => $userId
```

Comment:
Is this still userId or should it be actor

Reply:
actor. Fixed.